var/spack/repos/builtin/packages/sublime-text/package.py

```diff
@@ -39,8 +40,12 @@ def url_for_version(self, version):
             return "https://download.sublimetext.com/sublime_text_{0}_build_{1}_x64.tar.bz2".format(version[0], version[-1])
 
     def install(self, spec, prefix):
-        # Sublime text comes as a pre-compiled binary.
         install_tree('.', prefix)
```

Comment:
Is is possible to install everything directly in `prefix.bin`? What other files are in the root directory? I guess it might break if it tries to find things in different locations.

Reply:
There are some unnecessary files like `changelog.txt`, `sublime_text.desktop` or icons, which should not be part of PATH. Also there are some files, which I think are necessary if you want to develop packages for sublime (like `python3.3.zip` or `plugin_host`) but not to run sublime_text itself. So far I've not experienced any drawbacks running sublime without those files on PATH